sdk/digitaltwins/azure-digitaltwins-core/src/test/resources/session-records/QueryTests.validQuerySucceeds[1].json

```diff
@@ -1,179 +1,1593 @@
 {
```

Comment:
The file names are strange aren't they? [1]? when did that happen? do you know why?

Reply:
I'm not sure why its happening, but yeah I noticed that last week. They still seem to work fine though

Reply:
It is an array of files, duh.